frontend/src/ui/pages/HomePage.tsx

```diff
@@ -2,11 +2,14 @@ import { useEffect, useState } from "react"
 import Cookies from 'js-cookie';
 
 export function HomePage() {
-    let [nickname, setNickname] = useState("");
+    const [nickname, setNickname] = useState("");
 
     useEffect(() => {
-        const token = Cookies.get("access_token")
-        const url = `http://localhost:8004/users/me?token=${encodeURIComponent(token)}`;
+        const token = Cookies.get("refresh_token")
+        if (!token) {
+            return
+        }
+        const url = `/api/v1/users/me?token=${encodeURIComponent(token)}`;
```

Comment:
pass token in authorization header (front branch)

Reply:
Je vous laisse gerer ca, en effet le token doit etre dans le header